app/scripts/controllers/serviceInstance.js

```diff
@@ -0,0 +1,120 @@
+                                                     ProjectsService,
+                                                     ServiceInstancesService) {
+    $scope.alerts = {};
+    //$scope.displayName = null;
+    $scope.projectName = $routeParams.project;
+    $scope.serviceInstance = null;
+    $scope.serviceClass = null;
+    $scope.serviceClasses = null;
+
+    $scope.breadcrumbs = [
+      {
+        title: "Provisioned Services",
+        link: "project/" + $routeParams.project + "/browse/service-instances"
+      }
+    ];
+
+    $scope.deprovision = function() {
+      ServiceInstancesService.deprovision($scope.serviceInstance);
+    };
+
+    var watches = [];
+
+    /*var setDisplayName = function() {
+      if(!$scope.serviceInstance || !$scope.serviceClasses) {
+        return;
+      }
+
+      $scope.displayName = $filter('serviceInstanceDisplayName')($scope.serviceInstance, $scope.serviceClasses);
+    };*/
+
+    var updateBreadCrumbs = function() {
+      $scope.breadcrumbs.push({
+        title: $filter('serviceInstanceDisplayName')($scope.serviceInstance, $scope.serviceClasses)
+      });
+    };
+
+    var updateServiceClassMetadata = function() {
+      if(!$scope.serviceInstance || !$scope.serviceClasses) {
+        return;
+      }
+
+      updateBreadCrumbs();
+
+      var serviceClassName = _.get($scope.serviceInstance.spec, 'serviceClassName');
+      $scope.serviceClass = _.get($scope.serviceClasses, [serviceClassName]);
+      $scope.plan = _.find(_.get($scope.serviceClass, 'plans'), {name: $scope.serviceInstance.spec.planName });
+    };
+
+    var serviceResolved = function(service, action) {
+      $scope.loaded = true;
+      $scope.serviceInstance = service;
+
+      if (action === "DELETED") {
+        $scope.alerts["deleted"] = {
+          type: "warning",
+          message: "This provisioned service has been deleted."
+        };
+      }
+
+      //setDisplayName();
+      updateServiceClassMetadata();
+    };
+
+    ProjectsService
+      .get($routeParams.project)
+      .then(_.spread(function(project, context) {
+        $scope.project = project;
+        $scope.projectContext = context;
+
+        DataService
+          .get({
+            group: 'servicecatalog.k8s.io',
+            resource: 'instances'
+          }, $routeParams.instance, context, { errorNotification: false })
+          .then(function(service) {
+
+            serviceResolved(service);
+
+            /*watches.push(DataService.watchObject({
+              group: 'servicecatalog.k8s.io',
+              resource: 'instances'
+            }, $routeParams.instance, context, serviceResolved));*/
+            /*watches.push(DataService.watch({
+              group: 'servicecatalog.k8s.io',
+              resource: 'bindings'
+            }, context, function(bindings) {
+              $scope.bindings = _.filter(bindings.by('metadata.name'), { spec: { instanceRef: { name: instance.metadata.name } } });
+            }));*/
```

Comment:
Any reason this block is commented out? We should watch the objects for live page updates.

Reply:
Nope, just a test that wasn't reverted. Can do